flex/07-flex-layout-2/solution/solution.css

```
@@ -73,11 +73,11 @@ a {
   padding: 32px;
```

```suggestion
  padding: 48px;
```
The container padding will also need increasing to account for the lack of "outer" 16px spacing if no child margin.

---

increased padding to `48px`.